src/python/pants/init/engine_initializer.py

```diff
@@ -371,6 +374,19 @@ def union_membership_singleton() -> UnionMembership:
     def build_root_singleton() -> BuildRoot:
       return BuildRoot.instance
 
+    @rule
+    def single_build_file_address(specs: Specs) -> BuildFileAddress:
+      build_file_addresses = yield Get(BuildFileAddresses, Specs, specs)
+      if len(build_file_addresses.dependencies) != 1:
+        provenanced_addresses = yield Get(ProvenancedBuildFileAddresses, Specs, specs)
+        matched_addrs = [addr.build_file_address.to_address() for addr in provenanced_addresses]
+        output = '\n '.join(f':{addr}' for addr in matched_addrs)
```

Comment:
The leading colon on each line seems harmful to cut/paste. If the addr is foo/bar, the printed line will be `:foo/bar` and that is highly confusing since that line taken in total as literal _is_ a valid address and it points to a target named 'foo/bar' in a root BUILD file.I'd kill the leading colons or use a different bullet if you were after a bullet character. And even then, pad a space before the actual address.

Reply:
That was just a particularly-boneheaded end-of-day-Friday typo, removed :)